src/components/Control/Checkbox/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { MouseEvent } from 'react';
```

Comment:
```suggestion
import React from 'react';
import type { MouseEvent } from 'react';
```
빌드 결과물에 타입 관련 정보가 들어가지 않도록 이렇게 나눠서 써주는게 어떨까요?

Reply:
PopoverItem에서 사용하는 MouseEventHandler도 동일하게 type import로 변경하였습니다